app/policies/case_group_policy.rb

```diff
@@ -0,0 +1,42 @@
+class CaseGroupPolicy < ApplicationPolicy
+  class Scope < ApplicationPolicy::Scope
+    def resolve
+      case user
+      when CasaAdmin, Supervisor
+        scope.where(casa_org: @user.casa_org)
+      when Volunteer
+        scope.none
+      else
+        scope.none
```

Comment:
Some policies raise "user type unrecognized" or similar here. I think it makes more sense just to not return the records.

I almost added AllCasaAdmin to the cases, but currently they fall here.

Reply:
Yeah, come to think of it, I don't think I've seen it used outside of a PolicyScope. But it's nice to have an empty AR collection instead of [] here.

Thanks for the AllCasaAdmin explanation, that helps. So they should definitely get `scope.none`'d by default.